OsmAnd/res/values/strings.xml

```diff
@@ -318,33 +318,33 @@
     <string name="av_def_action_choose">On request</string>
-    <string name="audio_video_notes_desc">Provides quick access to start/stop taking an audio, video, or photo note.</string>
-    <string name="bearing_desc">Shows the absolute or relative bearing to a target in angular units (in 180 degree, 360 degree, or milliradians mode).</string>
-    <string name="navigation_points_desc">Navigational widgets are enabled during navigation to display information such as distance, arrival or remaining time, next turns, bearing, current/next street name, lane information, max speed, approaching alerts, POIs, waypoints.</string>
+    <string name="audio_video_notes_desc">Provides quick access to start/stop taking a recording (audio, video or photo).</string>
+    <string name="bearing_desc">The bearing widget shows the absolute or relative bearing to a target in angular units (degrees 180, degrees 360, milliradians).</string>
+    <string name="navigation_points_desc">Navigational widgets during navigation to display info such as distance, arrival or time left, the next turns, bearing, current street name, lane info, max speed, approaching alerts, POIs, waypoints.</string>
     <string name="navigation_points">Navigation points</string>
-    <string name="route_maneuvers_desc">Navigational widgets are enabled during navigation to display information such as distance, arrival or left time, next turns, bearing, current street name, lanes information, max speed, approaching alerts, POIs, waypoints.</string>
+    <string name="route_maneuvers_desc">Navigational widgets are enabled during navigation to display info such as distance, arrival or left time, next turns, bearing, current street name, lane info, max speed, approaching alerts, POIs, waypoints.</string>
     <string name="route_maneuvers">Route maneuvers</string>
-    <string name="get_altitude_information">Get elevation information</string>
-    <string name="calculating_altitude">Calculating elevation</string>
+    <string name="get_altitude_information">Get elevation info</string>
+    <string name="calculating_altitude">Looking up elevation…</string>
     <string name="calculate_online">Calculate online</string>
-    <string name="track_attach_to_the_roads_descr">The track will be attached to the roads using the offline maps, the track geometry will be changed.</string>
+    <string name="track_attach_to_the_roads_descr">Attaches the track to roads using the offline maps, changing the track geometry.</string>
     <string name="get_altitude_data">Get elevation data</string>
     <string name="calculate_altitude">Calculate elevation</string>
     <string name="no_altitude_data">No elevation data</string>
     <string name="shared_string_speed">Speed</string>
     <string name="app_mode_mountain_bicycle">Mountain bike</string>
     <string name="turn_on_public_transport_description">Turn on to see public transport routes on the map.</string>
     <string name="release_4_2">
```

Comment:
This string is outdated.

Reply:
Moving them all to fastlane files or their own file would be great.
There was an &amp; here, and now there is none of that going on.